installer/scripts/omsadmin.sh

```diff
@@ -236,10 +232,6 @@ parse_args()
         a)
             AZURE_RESOURCE_ID=$OPTARG
             ;;
-        c) 
-            COLLECTD=1
```

Comment:
Same printout idea here: print an informational message like "CollectD can no longer be configured with the --collectd option. Please see OMSAgent-for-Linux documentation here: https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/docs/OMS-Agent-for-Linux.md"